samples/createInput.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2022, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(projectNumber, location, inputId) {
+  // [START livestream_create_input]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // projectNumber = '012345678';
```

Comment:
As I know, we use `projectId` for the libraries. Not sure if this is same with `projectId` or different variable.

Reply:
As I understand it, both should work. Is it a best practice to use `projectId` in code samples? Our curl samples on the site use project number.

Reply:
Removed project number in favor or project ID.